templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/fragment_statement.html.twig

```diff
@@ -52,23 +52,25 @@
                 :init-priority-areas="JSON.parse('{{ selectedPriorityAreas|default([])|json_encode|e('js', 'utf-8') }}')"
                 init-fragment-text="{{ fragment.text|default|wysiwyg }}"
                 procedure-id="{{ procedure }}"
+                statement-text="{{ templateVars.statement.text|default|escape|wysiwyg(hasPermission('feature_obscure_text') ? 'dp-obscure' : []) }}"
                 inline-template>
                 <div>
                     <p class="u-mt-0_5">{{ "text.statement.fragment"|trans|wysiwyg }}</p>
 
                     {# statement text #}
                     <p class="weight--bold">
                         {{ "statement"|trans }}
-                        <dp-copy-paste-button
-                            button-label="statement.copy"
-                            class="float--right"
-                            copy-content="{{ templateVars.statement.text|default|escape|wysiwyg(hasPermission('feature_obscure_text') ? 'dp-obscure' : []) }}"
+                        <dp-button
+                            class="float--right btn--blank color-main font-size-medium"
+                            icon="file"
```

Comment:
The "copy" icon is not released yet but it will be, in the [next demosplan-ui release](https://github.com/demos-europe/demosplan-ui/blob/main/CHANGELOG.md#added).
```suggestion
                            icon="copy"
```

Reply:
Thanks, I changed it here 005e36862c408ece0a52c0d6bd71150210be9bfc and will wait for the demosplan-ui release